app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -11,94 +11,129 @@ import { useWhaleApiClient } from '../../contexts/WhaleContext'
 import { RootState } from '../../store'
 import { firstTransactionSelector, ocean, OceanTransaction } from '../../store/ocean'
 import { translate } from '../../translations'
+import * as Clipboard from 'expo-clipboard'
+import { CTransactionSegWit } from '@defichain/jellyfish-transaction/dist'
 
-async function handlePress (txid: string): Promise<void> {
-  // TODO(thedoublejay) explorer URL
-  const url = `https://explorer.defichain.io/#/DFI/mainnet/tx/${txid}`
-  const supported = await Linking.canOpenURL(url)
-  if (supported) {
-    await Linking.openURL(url)
-  }
+const MAX_AUTO_RETRY = 1
+
+// async function gotoExplorer (txid: string): Promise<void> {
```

Comment:
I think we can keep the redirect and store the txid in the clipboard momentarily.